lua/astrocommunity/completion/tabby-nvim/init.lua

```diff
@@ -0,0 +1,16 @@
+return {
+  "TabbyML/vim-tabby",
+  event = "User AstroFile",
```

Comment:
```suggestion
  event = "VeryLazy",
```

@PatMulligan This might be a better event to give it time to connect on startup